x-pack/plugins/apm/dev_docs/query_debugging_in_development_and_production.md

```diff
@@ -14,4 +14,147 @@ There will be an `_inspect` key containing every Elasticsearch query made during
 
 ![image](https://user-images.githubusercontent.com/209966/140500012-b075adf0-8401-40fd-99f8-85b68711de17.png)
 
+## Example 
 
+When "Inspect ES queries" are enabed all API calls to the APM API will be include the query param `_inspect=true`. For the environments API the request / response will be:
```

Comment:
Do we want anything here about using the inspector? You don't really need to look at the console/dev tools responses if you're using the inspector.

Reply:
I think having both is useful. In [this issue](https://github.com/elastic/apm-dev/issues/643) I mention the `_inspect` flag.

Documenting the UI Inspector tool would be a good addition.